index.js

```diff
@@ -14,21 +19,78 @@ const proxyHandler = {
   }
 }
 
-exports._register =  function _register(name, namespace, set) {
+function _register(name, namespace, set, specifier) {
```

Comment:
Could this be moved to a different file and resolved relatively instead of exposing a private function?

Reply:
Yes, I was thinking that ought to be done.